spring-cloud-gcp-data-spanner/src/test/java/com/google/cloud/spring/data/spanner/repository/support/SpannerRepositoryImplTests.java

```diff
@@ -264,6 +265,20 @@ public void deleteByIdTest() {
 		verify(this.template, times(1)).delete(Object.class, A_KEY);
 	}
 
+	@Test
+	public void deleteAllByIdTest() {
+		List<Key> unconvertedKey = Arrays.asList(Key.of("key1"), Key.of("key2"));
+
+		when(this.entityProcessor.convertToKey(Key.of("key1"))).thenReturn(Key.of("key1"));
```

Comment:
(Probably not directly related to this PR) This `convertToKey` behavior seems strange. It's takes a `Key` and returning `Key`. I see it's not converting anything. Do you know why?

Reply:
This `convertToKey` method itself can take in a `Key`, a single key component, or an array of key components to convert. You are right it's not doing anything in this case.  I tried to keep it simple and consistent with other tests as it's not the point of the test.

Reply:
Got it. Thanks.

Reply:
hmm, it does feel less odd this way. Let me make the change.